code/game/gamemodes/endgame/xmas/krampus.dm

```diff
@@ -28,12 +28,60 @@
 	real_name = "Krampus"
 	status_flags = GODMODE|CANPUSH
 
-	New(var/new_loc)
-		h_style = "Bald"
-		..(new_loc, "Krampus")
-		maxHealth=999999
-		health=999999
+/mob/living/carbon/human/krampus/New(var/new_loc)
+	h_style = "Bald"
+	..(new_loc, "Krampus")
+	maxHealth=999999
+	health=999999
+	var/obj/item/weapon/krampus/sack = new /obj/item/weapon/krampus(src)
+	put_in_hands(sack)
 
+/mob/living/carbon/human/krampus/proc/SackEm(var/mob/M)
```

Comment:
There are, depending on how you count, five or six different cases just in `unsorted.dm`, the first file that happened to be open in my text editor,and one that contains many very important procs